src/components/ProblemList/ProblemList.tsx

```diff
@@ -16,14 +16,16 @@ interface ProblemListProps {
     className?: string;
     style?: CSSProperties;
     isSmallSize?: boolean;
-    interviewId?: number;
+    interviewId: number;
```

Comment:
Problem list is used at the /problems where you don't have any interviews or sections